subset/switches/test_switch

```diff
@@ -8,8 +8,17 @@ if [ -n "$LOCAL_IP" ]; then
     ip addr add $LOCAL_IP dev `hostname`-eth0
     echo Switch test with port $SWITCH_PORT.
     ping -n -c 10 $SWITCH_IP
-    java -jar switches/target/switchtest-0.0.1-jar-with-dependencies.jar $SWITCH_IP $SWITCH_PORT
+    if[-n "$POE_ENABLED"]; then
+        java -jar switches/target/switchtest-0.0.1-jar-with-dependencies.jar $SWITCH_IP $SWITCH_PORT $POE_ENABLED
+    else
```

Comment:
@ewRedstone I'm not convinced you need an if/else here, just rely on $POE_ENABLED being set either true or false by DAQ

Reply:
Agreed an "if" is not the most elegant way to do this.  If anything, the "if" should just set the default value. You can also use the ${XXXX:-DEFAULT} syntax of BASH to have a default value.  See top of bin/setup_dev for some examples.

Also, this env variable will not be set by daq -- it's going to be delivered as a module_config.json.  See test_nmap for an example.

Reply:
im scrapping this idea, and making the poe element fully automated inside the java code

Reply:
@ewRedstone In this context, the poe test should be turned `on` or `off` by the operator via module_config.json as Trevor says (thanks Trever!). 

A fully automated 'is this switch poe, therefore run this part of the swtich test' _is_ useful if there is user (read, operator) error but it's likely simpler for now to read the POE_ENABLED from a module_config.json in the following form:

```
{
 "modules": {
   "bacext": {
     "enabled": true
   },
   "brute": {
     "enabled": true
   },
   "macoui": {
     "enabled": true
   },
   "nmap": {
     "enabled": true
   },
   "pass": {
     "enabled": true
   },
   "switch": {
     "enabled": true,
     "poe": {
       "support": "false"
     }
   },
   "tls": {
     "enabled": true
   }
 },
 "process": {
   "approver": "ed",
   "operator": "link"
 }
}
```

Reply:
poe is now automated